ts/src/connections/relayTunnelConnector.ts

```diff
@@ -123,8 +123,12 @@ export class RelayTunnelConnector implements TunnelConnector {
             disconnectReason = SshDisconnectReason.connectionLost;
             error = undefined;
             try {
-                stream = await this.tunnelSession.createSessionStream(cancellation);
-                await this.tunnelSession.configureSession(stream, isReconnect, cancellation);
+                const streamAndProtocol = await this.tunnelSession.createSessionStream(
```

Comment:
Nit: can we decompose to stream and protocol in place?

Reply:
The `stream` is already defined, while `protocol` is not, so I found a destructuring assignment to be just as awkward as this.